src/services/doppler-legacy-client.ts

```diff
@@ -26,7 +26,8 @@ interface PayloadWithCaptchaToken {
 export type LoginErrorResult =
   | { blockedAccountNotPayed?: false; accountNotValidated?: false }
   | { blockedAccountNotPayed: true; accountNotValidated?: false }
-  | { accountNotValidated: true; blockedAccountNotPayed?: false };
+  | { accountNotValidated: true; blockedAccountNotPayed?: false }
+  | { accountIsCanceled: true; blockedAccountNotPayed?: false };
```

Comment:
I know that it is not the best, but could you add the new error flag in all error result types?

Something like:

```
{ blockedAccountNotPayed?: false; accountNotValidated?: false; accountIsCanceled?: false }
```

It will help in intellisense.